assembl/views/discussion/views.py

```diff
@@ -175,25 +175,28 @@ def home_view(request):
     return response
 
 
+@view_config(route_name='new_home', request_method='GET', http_cache=60,
+             renderer='assembl:templates/index_react.jinja2')
+@view_config(route_name='new_styleguide', request_method='GET',
+             renderer='assembl:templates/index_react.jinja2')
+@view_config(route_name='new_profile', request_method='GET',
+             renderer='assembl:templates/index_react.jinja2')
```

Comment:
Yeah, I think we just need two fallback routes that match all urls /{discussion}/v2/*reminder and /*reminder which render templates/index_react.jinja2 and you can remove 
new_styleguide and new_profile here.